src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -348,6 +358,9 @@ private void applyValidationType(VariantContext vc, Allele reportedRefAllele, Al
                 if (!rsIDs.isEmpty()) {
                     vc.extraStrictValidation(reportedRefAllele, observedRefAllele, rsIDs);
                 }
+                //default case - do as many validations as possible given no command line args
```

Comment:
What if rsIDs is not empty? Then it looks like you will call vc.validateAlternatieAlleles() in two places. What if rsIDs is empty the the reference isn't empty? Where does that get called in the default case?